features/karaf/openhab-addons/src/main/feature/feature.xml

```diff
@@ -17,6 +17,19 @@
 
     <repository>mvn:org.openhab.core.features.karaf/org.openhab.core.features.karaf.openhab-core/${project.version}/xml/features</repository>
 
+    <!-- Features for Java EE bundles -->
+
+    <feature name="openhab-jaxb" description="JAXB bundles" version="1.0.0">
+        <bundle dependency="true">mvn:javax.xml.bind/jaxb-api/2.3.1</bundle>
+        <bundle dependency="true">mvn:com.sun.xml.bind/jaxb-impl/2.3.2</bundle>
+        <bundle dependency="true">mvn:com.sun.xml.bind/jaxb-core/2.3.0</bundle>
+    </feature>
+    <feature name="openhab-jaxws" description="JAX-WS (SOAP) bundles" version="1.0.0">
+        <bundle dependency="true">mvn:javax.xml.soap/javax.xml.soap-api/1.4.0</bundle>
+        <bundle dependency="true">wrap:mvn:com.sun.xml.ws/jaxws-rt/2.3.2$Bundle-Name=JAX%20WS%20RI%20Runtime%20Bundle&amp;Bundle-SymbolicName=com.sun.xml.ws.jaxws-rt&amp;Bundle-Version=2.3.2</bundle>
```

Comment:
Because this is a wrapped bundle this feature also requires a "wrap" feature prerequisite.